src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity
+public class Product {
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @NotNull
+    private String name;
+
+    private String description;
```

Comment:
Domyślnie zostanie ustawiona długość kolumny na 255 znaków, czy to nie za mało jak na opis ?